src/main/thrift/parquet.thrift

```diff
@@ -270,8 +270,11 @@ struct Statistics {
     * may set min_value="B", max_value="C". Such more compact values must still be
     * valid values within the column's logical type.
     *
-    * Values are encoded using PLAIN encoding, except that variable-length byte
-    * arrays do not include a length prefix.
+    * Values are encoded using PLAIN encoding, except that:
+    * 1) variable-length byte arrays do not include a length prefix.
+    * 2) geometry logical type with BoundingBoxOrder uses max_value/min_value pair
```

Comment:
This is great to have as part of statitics, but trying to wrap my head around how writer implementations in WKB case can get min_value/max_value, not sure if they have the bounding box easily unless they deserialze.

Reply:
You're right. As you can see that `max_value` and `min_value` here are of `binary` type, which is the serialized form and cannot be consumed directly. AFAIK, the [C++](https://github.com/apache/arrow/blob/1e3772cac5f45edb6ada3d20140b77cc86208346/cpp/src/parquet/statistics.h#L286-L290) and [Java](https://github.com/apache/parquet-mr/blob/c241170d9bc2cd8415b04e06ecea40ed3d80f64d/parquet-column/src/main/java/org/apache/parquet/column/statistics/DoubleStatistics.java#L133-L139) Parquet implementations have provided functions to access the deserialized values from the `Statistics` class. We can also define an easy-to-use bounding box class and get it from `Statistics` implementation, but that's not the topic of the specs.

Reply:
> trying to wrap my head around how writer implementations in WKB case can get min_value/max_value

We have two options here:
1. If the input data is in a well-defined geometry object, collecting bounding box should be easy but the interface of the parquet library would be complicated.
2. The parquet writer only accepts WKB-encoded binary data, then the writer is required to deserialize values to get the coordinates which degrade the performance.

Reply:
Yes, option 1 is more efficient but option 2 might be easier for different parquet impls. parquet-mr (which is the java impl of parquet) will depend on JTS and it is pretty natural to accept JTS Geometry as input data. However, for other parquet impls (e.g. parquet-cpp from arrow cpp, or parquet rust from arrow-rs), perhaps we need to leverage GeoArrow?

cc @pitrou @mapleFU @tustvold @zeroshade @etseidl to get awareness of this.

Reply:
@szehon-ho 

I think the native encoding is no longer really needed if the writer can do bbox itself.

Both option 1 and 2 sound fine to me. And, if the performance is NOT degraded too much, maybe Option 2 is even a better choice in terms of adoption?

CC @zhangfengcdt @Kontinuation